contributors/devel/sig-architecture/staging.md

```diff
@@ -4,13 +4,14 @@ The [staging/ directory](https://git.k8s.io/kubernetes/staging) of Kubernetes co
 
 We publish the staging repos using the [publishing bot](https://git.k8s.io/publishing-bot). It uses `git filter-branch` essentially to [cut the staging directories into separate git trees](https://de.slideshare.net/sttts/cutting-the-kubernetes-monorepo-in-pieces-never-learnt-more-about-git) and pushing the new commits to the corresponding real repositories in the [kubernetes organization on Github](https://github.com/kubernetes).
 
-The list of staging repositories and their published branches are listed in [publisher.go inside of the bot](https://git.k8s.io/publishing-bot/cmd/publishing-bot/publisher.go). Though it is planned to move this out into the k8s.io/kubernetes repository.
+The list of staging repositories and their published branches are listed in [rules.yaml inside of the k8s.io/kubernetes repository](https://git.k8s.io/kubernetes/staging/publishing/rules.yaml).
 
 At the time of this writing, this includes the branches
 
 - master,
-- release-1.8 / release-5.0,
-- and release-1.9 / release-6.0
+- release-1.15 / release-12.0,
```

Comment:
We still have rules for release-1.14 and sync it...

Reply:
1.14 uses godeps instead of gomodules, so we have the rules listed in a separate yaml file: https://github.com/kubernetes/kubernetes/blob/master/staging/publishing/rules-godeps.yaml

Reply:
Oh, ok then :)